### ISSUE TITLE
Add the customer id to the cart

### ISSUE THREAD

**Issue by zanatoshi** [bigcommerce/storefront-data-hooks#84]
Hi guys,
thank you for this easy to use hooks.

When the addItem hook create the cart, it would be nice to add the customer id if the user is connected.

I guess we could just get the customer_id in the useAddItem hook then use it as a param to the api call.

Let me know if this is something you may want to change.

**Comment by jorgemasta** [comment]
Good idea. Necessary steps:

1. Get (if exists) the the `customerToken`
```js
// src/api/cart/index.ts
const customerToken = req.cookies[config.customerCookie]
```
2. Add the `customerToken` to the `body` sent to handlers
```js
// src/api/cart/index.ts
const body = { cartId, include, customerToken }
```
3. Get the `customerId` from the `customerToken`
```js
// src/api/cart/handlers/add-item.ts
const customerId = customerToken && (await getCustomerId({ customerToken, config }))
```
4. Add (if exists) the `customerId` to the request body

**Comment by jorgemasta** [comment]
It will be released in `1.6.0`
